### DISCUSSION TITLE
Dropzone in windows having issues while opening the file picker

### DISCUSSION THREAD

**Discussion by Rohithgilla12** [mantinedev/mantine#1932] [Feature requests]
First, thanks for the amazing work on this library.

I don't know where to report this bug, so reporting it here.

When trying to use `accept` parameter with the dropzone the file picker it opens doesn't show all the custom types.

But it does work when we try to drag and drop the file.
### Code
```tsx
const SUPPORTED_FILE_TYPES = [MIME_TYPES.jpeg, MIME_TYPES.png, MIME_TYPES.webp];

<MantineDropzone
  accept={SUPPORTED_FILE_TYPES}
// other props

```

![image](https://user-images.githubusercontent.com/19389850/181903035-ad0da4f3-3814-44e7-b8c4-588f03d6f271.png)

As you see in the bottom right corner, the file type is only one. But when clicked on the dropdown it allows to change the file type filter.

**Comment by rtivital** [comment]
I guess this is how react-dropzone works with Windows file picker, not much can be done on our side.
